chapters/cli/exit.rst

```diff
@@ -1,27 +1,29 @@
 ====================
 Gracefully Exit Jina
 ====================
-In this section, you will learn best practices to shut down a Flow and exit Jina correctly.
+
+In this section, you will learn best practices shutting down a Flow and exiting Jina correctly.
 
 .. contents:: Table of Contents
     :depth: 4
 
 Feature description and expected outcome
 ----------------------------------------
-Jina provides several methods to exit gracefully.
-Through these ways the Flow will terminate and release all resources.
+Jina provides several methods to exit this service gracefully. In this way, the Flow is terminated and all occupied resources are released.
 
 Before you start
 -----------------
-Make sure you install Jina via `Installation <https://docs.jina.ai/chapters/install/os/index.html>`_.
+Make sure that Jina is installed properly as explained in `Installation <https://docs.jina.ai/chapters/install/os/index.html>`_. Define a Flow as explained in `101: Basic components <https://docs.jina.ai/chapters/101/>` , and `102: How basic components work together <https://docs.jina.ai/chapters/102/>`_.
```

Comment:
In my view, it sounds very direct to start a sentence with Define, I would rather say. 

and you had an active running Flow correctly set up as explained in 101 and 102.

Reply:
a minor comment, can we change the fixed link to the relative link, such as `:doc:chapters/install/os/index` (this might be incorrect)

Reply:
@FionnD : I will correct that -- your text version sounds way more friendly
@bwanglzu : I saw the absolute links, and will replace them by relative links